### ISSUE TITLE
Error writing to parquet using kerchunk

### ISSUE THREAD

**Issue by TomNicholas** [TomNicholas/VirtualiZarr#115]
@jsignell it seems the parquet writer implementation in #72 is more fragile then it looked - I tried it in [this notebook](https://gist.github.com/TomNicholas/8b4aaf464fe130d5c16aae97f4d18c50) and got an error from `fsspec`. I'm not sure if this is `fsspec`'s fault but it would be nice to make it a little more robust if possible.

**Comment by jsignell** [comment]
It looks like you have some alternative version of kerchunk. Can you confirm that you have 0.2.5 installed?

**Comment by jsignell** [comment]
In particular your traceback looks like it is pointing to these lines:

```python
    out = LazyReferenceMapper.create(
        record_size, root=url, fs=fs, categorical_threshold=categorical_threshold
    )
```
https://github.com/fsspec/kerchunk/blob/3c4e9fc960e159875e8f258ccd20fdbc565513df/kerchunk/df.py#L154-L156

when the current version looks more like: 

```python
    out = LazyReferenceMapper.create(
        record_size=record_size,
        root=url,
        fs=fs,
        categorical_threshold=categorical_threshold,
    )
```
https://github.com/fsspec/kerchunk/blob/0.2.5/kerchunk/df.py#L154-L159

**Comment by TomNicholas** [comment]
You're right! Upgrading to `kerchunk==0.2.5` fixed it. Sorry for the noise @jsignell .
